src/vmm/src/rpc_interface.rs

```diff
@@ -908,11 +928,18 @@ mod tests {
             Ok(())
         }
 
-        pub fn set_mmds_config(&mut self, _: MmdsConfig, _: &str) -> Result<(), MmdsConfigError> {
+        pub fn set_mmds_config(
+            &mut self,
+            mmds_config: MmdsConfig,
+            _: &str,
+        ) -> Result<(), MmdsConfigError> {
             if self.force_errors {
                 return Err(MmdsConfigError::InvalidIpv4Addr);
             }
-            self.mmds_set = true;
+            let mut mmds_guard = self.mmds.lock().unwrap();
```

Comment:
Same as above, I think an `expect()` with error message would be better here instead of plain `unwrap()`.

Reply:
since this is defined in the tests module, doesn't really matter. I did however replace the unwrap with expect in the other places 👍🏻